bazel/boringssl_static.patch

```diff
@@ -1,8 +1,16 @@
 diff --git a/BUILD b/BUILD
-index cfa695a44..e748f9b46 100644
+index 1ec2bdf18..8834f80c2 100644
 --- a/BUILD
 +++ b/BUILD
-@@ -101,7 +101,10 @@ linux_copts = posix_copts + [
+@@ -96,12 +96,18 @@ linux_copts = posix_copts + [
+     # it should not be set on Apple platforms, where it instead disables APIs
+     # we use. See compat(5) and sys/cdefs.h.
+     "-D_XOPEN_SOURCE=700",
++    "-Wno-array-bounds",
```

Comment:
in wondering if we should add a TODO to remove with a link to upstream issue

Reply:
added in thew patch to make it clear it's about these 3